api/eid_server/eid_types/sessionid.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from numpy import random
+import binascii
+
+
+class SessionID:
+
+    def __init__(self):
+        self.__sessionid = ""
+
+    @property
+    def id(self):
+        i = 16
+        while len(self.__sessionid) < 32:
+            a = random.bytes(i)
```

Comment:
numpy.random is not a cryptographically safe way to generate random numbers

Reply:
why don't you just generate 16 bytes of random data?

Also, you could use [random.choice](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.random.choice.html#numpy.random.choice)? (After seeding it with crypto-safe random numbers.)